docs/StructureAndRationale.md

```diff
@@ -0,0 +1,75 @@
+# CTK/CTS Design - Structure and Rationale
+
+## Design Goals
+
+The intent of the CTK is to make it easy to run and write GA4GH API-centric tests, and to use the test results in various reporting environments.
+
+## Layout Overview
+
+The CTK is structured as a set of Maven modules:
+```
+                                                             +-----------------------------+
+ctk bash script, raw java invoke +--------------------------->ctk-cli                      |
```

Comment:
I understand that`ctk-server` was something of an experiment, and as such was largely kept under wraps.  You might notice that the module contains configuration files and other items that duplicate parts of the CLI; these await either a grand refactoring or a mass purge.

As you know, we've been emphasizing running the CTK from the command line.  I'd like to continue that and consider the future of `ctk-server` a little further down the line.